tests/pages/services/ServiceFormModal-cy.js

```diff
@@ -258,6 +258,41 @@ describe('Service Form Modal', function () {
       cy.visitUrl({url: '/services/overview/%2F/create'});
     });
 
+    it('should fill the entire viewport', function () {
+      var isModalFullScreen = true;
+
+      cy.window()
+        .then(function ($window) {
+          cy.get('.modal-full-screen')
+            .should(function ($element) {
+              if ($element[0].clientHeight !== $window.innerHeight || $element[0].clientWidth !== $window.innerWidth) {
+                isModalFullScreen = false;
+              }
+              expect(isModalFullScreen).to.be.equal(true);
+            });
+        });
+    });
+
+    it('should be horizontally and vertically centered in the modal container', function () {
+      cy.get('.modal-body-wrapper')
+        .should(function ($modalWrapper) {
+          const modalWrapperRect = $modalWrapper[0].getBoundingClientRect();
+          const modalWrapperMidpointY = Math.abs(modalWrapperRect.top + (modalWrapperRect.height / 2));
+          const modalWrapperMidpointX = Math.abs(modalWrapperRect.left + (modalWrapperRect.width / 2));
+
+          const $modalContent = $modalWrapper.find('.row.panel-grid');
+          const modalContentRect = $modalContent[0].getBoundingClientRect();
+          const modalContentMidpointY = Math.abs(modalContentRect.top + (modalContentRect.height / 2));
+          const modalContentMidpointX = Math.abs(modalContentRect.left + (modalContentRect.left / 2));
+
+          const topPosDifference = Math.abs(modalWrapperMidpointY - modalContentMidpointY);
+          const leftPosDifference = Math.abs(modalWrapperMidpointX - modalContentMidpointX);
+
+          expect(topPosDifference <= modalContentMidpointY).to.be.equal(true);
+          expect(leftPosDifference <= modalContentMidpointX).to.be.equal(true);
```

Comment:
I think we are still comparing the wrong values here. In an ideal world, `topPosDifference` and `leftPosDifference` would be `0`. But it's acceptable for the mid-points to be within ~5 pixels of one another. So instead of comparing `topPosDifference <= modalContentMidpointY` and `leftPosDifference <= modalContentMidpointX` we should be comparing `leftPosDifference <= 5` and `leftPosDifference <= 5`. The value `5` represents the alignment threshold that we consider acceptable.

I defined an `alignmentTheshold` variable elsewhere in this file (for the test `Should vertically align the placement constraint delete row button`), so we could move that variable's declaration and reuse it here.